go/internal/infra/local.go

```diff
@@ -0,0 +1,67 @@
+package infra
+
+import (
+	"github.com/feast-dev/feast/go/internal/config"
+	"github.com/feast-dev/feast/go/protos/feast/core"
+	"github.com/golang/protobuf/proto"
+	"github.com/google/uuid"
+	timestamppb "google.golang.org/protobuf/types/known/timestamppb"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+type LocalRegistryStore struct {
+	filePath string
+}
+
+func NewLocalRegistryStore(config *config.RegistryConfig, repoPath string) *LocalRegistryStore {
+	lr := LocalRegistryStore{}
+	registryPath := config.Path
+	if filepath.IsAbs(registryPath) {
+		lr.filePath = registryPath
+	} else {
+		lr.filePath = filepath.Join(repoPath, registryPath)
+	}
+	return &lr
+}
+
+func (r *LocalRegistryStore) GetRegistryProto() (*core.Registry, error) {
+	registry := &core.Registry{}
+	// Read the local registry
+	in, err := ioutil.ReadFile(r.filePath)
+	// Use an empty Registry Proto if file not exists or parse Registry Proto content from file
+	if err != nil {
+		return nil, err
+	}
+	if err := proto.Unmarshal(in, registry); err != nil {
+		return nil, err
+	}
+	return registry, nil
+}
+
+func (r *LocalRegistryStore) UpdateRegistryProto(rp *core.Registry) error {
+	return r.writeRegistry(rp)
+}
+
+func (r *LocalRegistryStore) Teardown() error {
```

Comment:
I had this to comply with RegistryStore interface. I don't know if the plan is to migrate every feature from python sdk to go server, then I've made this interface in case we want to add more Registry Store (gcp, s3). @woop ?

Reply:
We dont need teardown. We need the minimal amount of code to read from an online store.

Reply:
this is a registry store and this teardown is used to remove registry.db or remove gcp, s3 resources. Should teardown be removed from RegistryStore interface?

Reply:
We should remove it from this interface since it's not expected to be called by anyone. If we need to we can add it later. 

The go code should basically only read from the the feature store or registry - not update or destroy it IMO.